src/test/test-bootspec.c

```diff
@@ -8,7 +8,6 @@
 #include "tmpfile-util.h"
 
 TEST_RET(bootspec_sort) {
-
         static const struct {
```

Comment:
Uh, I actually prefer that we add a newline here...

Reply:
We don't put an empty line between a one-line function signature and the variable declaration block.

Reply:
We often put `static const` structs/arrays at the beginning of the function, and insert a newline regardless of function parameters. I think that way is nicer, but feel free to disagree and merge this.

Reply:
I dropped that last commit. This was a trivial change, and it doesn't seem worth to do it if there is no universal agreement.